### PR TITLE
Fix example path in readme file.

### PR DESCRIPTION
**Pull request by lonegunmanb**
The example path in the readme file is incorrect now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ provider "kubernetes" {
 }
 ```
 
-To try the module, please run `terraform apply` command in `test/fixture` folder.
+To try the module, please run `terraform apply` command in `examples`'s sub folder.
```
Comment (zioproto) [inline README.md:125]: I would rephrase this way:

Some examples to try the module, are available in the `examples` folder. These examples are tested against every PR with the [E2E test](https://github.com/Azure/terraform-azurerm-aks/blob/master/test/e2e/terraform_aks_test.go).

File: README.md [hunk 2]
```diff
@@ -202,7 +202,7 @@ docker run --rm -v $(pwd):/src -w /src -e TF_VAR_key_vault_firewall_bypass_ip_ci
 ```
 
 On Windows Powershell:
-
+``
```
Comment (zioproto) [inline README.md:205]: Is this a typo ?

Comment (lonegunmanb) [reply README.md]: Thanks @zioproto for review this pr! Would you please give another review? Thanks.
